src/custom/components/OrdersPanel/index.tsx

```diff
@@ -51,6 +49,20 @@ const SideBar = styled.div`
   `};
 `
 
+const SidebarBackground = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 0;
+  width: 100%;
+  height: 100%;
+  background: rgb(0 0 0 / 25%);
```

Comment:
what is the / here? never seen

Reply:
RGB color with a percentage opacity (vs. RGBA (alpha setting opacity).